Game/Pregnancy/MenstrualCycle.gd

```diff
@@ -77,7 +86,11 @@ func getCharacter():
 func initCycle():
 	cycleProgress = RNG.randf_range(0.0, 1.0)
 	
-func newCycle():
+func newCycle(shouldClearNonPregEggs: = true):
+#	print(getCharacter().getName(), " Entered new cycle" )
+	if(shouldClearNonPregEggs):
+		for orificeType in OrificeType.getAll():
+			eggCells[orificeType] = []
```

Comment:
This is kinda unrealistic, why not keep the old behaviour and instead make it so you can't get into heat if you ovulated this cycle already. This would fix the problem with too many eggcells

Reply:
Unrealistic? But not fertilised eggs are shed during menstruation phase.